wifi-subsys/main/main.c

```diff
@@ -33,15 +33,18 @@
 #include "freertos/event_groups.h"
 #include "nvs_flash.h"
 #include "wifi.h"
-
+#include "httpsclient.h"
 static const char* TAG = "m4a-wifi-subsys";
 
 
 void app_main(void)
 {
 
-    printf("Init main m4a-subsys.\n");
-
+    ESP_ERROR_CHECK( nvs_flash_init() );
+  
     wifi_init();
+  
+    xTaskCreate(&https_request_task, "https_get_task", 8192, NULL, 5, NULL);
+    printf("Init main m4a-subsys.\n");
```

Comment:
don't touch main file, check your code with unit tests, you can  test you code in main after that tests you must let this file clean.

Reply:
yep